state_machine/applications/flight/Tasks/beacon_task.py

```diff
@@ -24,5 +24,10 @@ async def main_task(self):
         except Exception:
             try:
                 os.mkdir('/sd/logs/')
+                # make sure to still log file when making the directory or else we lose the first beacon_packet of data
+                beacon_packet = logs.beacon_packet(self)
+                file = open(f"/sd/logs/log{int(currTime//TIMEINTERVAL)}.txt", "ab+")
```

Comment:
I think this filename should include the word "beacon" somewhere

Reply:
same with the one above

Reply:
Sounds good! I have changed the files to be beacon_log instead of just log.